resources/models/resource.py

```diff
@@ -378,3 +392,33 @@ class Meta:
         verbose_name = _('resource image')
         verbose_name_plural = _('resource images')
         unique_together = (('resource', 'sort_order'),)
+
+
+class ResourceEquipment(ModifiableModel):
+    resource = models.ForeignKey(Resource, related_name='resource_equipments')
+    equipment = models.ForeignKey(Equipment, related_name='resource_equipments')
+    data = hstore.DictionaryField(null=True, blank=True)
```

Comment:
There is no concrete use case yet, but the idea is to enable flexible and fast filtering. We want to allow different departments in the City to set their own custom attributes, and they will then build their customized UIs on top of our API.